server/integration/event-logger/src/main/java/org/infinispan/server/eventlogger/ServerEventImpl.java

```diff
@@ -21,7 +22,7 @@
  * @since 8.2
  */
 @SerializeWith(ServerEventImpl.Externalizer.class)
-public class ServerEventImpl implements EventLog {
+public class ServerEventImpl implements EventLog, Serializable {
```

Comment:
I am guessing we aren't too worried about performance of the serialization of these?  Otherwise I would say we should add an Externalizer for it.